app/tasks/maintenance_tasks/csv_task.rb

```diff
@@ -3,9 +3,9 @@
 require 'csv'
 
 module MaintenanceTasks
-  # Base class that is inherited by the host application's Task classes for
+  # Module that is included into Task classes by Task#csv_collection for
   # processing CSV files.
-  class CsvTask < Task
+  module CsvTask
```

Comment:
Would a name like `CsvHandling` be more appropriate, since on its own this module isn't a `Task`?

Reply:
Yeah, or `CsvProcessing` to make use of the language we've already established with `Task#process`?

Reply:
`CsvProcessing` is weird because it's actually not doing the processing, but everything else 😄 

What about `CsvCollection` just like the `csv_collection` method?

Reply:
LOL true. `CsvCollection` actually came to my mind as well, I like that 👍